.travis.yml

```diff
@@ -77,3 +77,7 @@ script:
 
 notifications:
   email: false
+
+branches:
+  exclude:
+    - /^sils\//
```

Comment:
As @sils suggested in https://github.com/coala/coala-bears/pull/1289 , you need to add a newline before EOF